src/ReflectionMethod.php

```diff
@@ -38,6 +39,13 @@ class ReflectionMethod extends BaseReflectionMethod
     private $declaringClass;
 
     /**
+     * Name of method
+     *
+     * @var string
+     */
+    private $methodName;
```

Comment:
Any benefits of remembering user-provided `$methodName` VS using `\ReflectionMethod::getName()`?

Reply:
(As above:) `parent::getName()` is not initialized in all cases. Less to special case.